Make.config

```
@@ -46,9 +46,9 @@ IOS_PACKAGE_VERSION_BUILD=$(IOS_COMMIT_DISTANCE)
 IOS_PACKAGE_UPDATE_ID=$(shell printf "2%02d%02d%02d%03d" $(IOS_PACKAGE_VERSION_MAJOR) $(IOS_PACKAGE_VERSION_MINOR) $(IOS_PACKAGE_VERSION_REV) $(IOS_PACKAGE_VERSION_BUILD))
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=11.0
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_11.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode11.app/Contents/Developer
+XCODE_VERSION=11.1
+XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_11.1_GM_Seed.xip
+XCODE_DEVELOPER_ROOT=/Applications/Xcode11.1-GM.app/Contents/Developer
```

I think this should be `Xcode111-GM.app` at least that is what we did for Xcode 10 minors

```suggestion
XCODE_DEVELOPER_ROOT=/Applications/Xcode111-GM.app/Contents/Developer
```